### PR TITLE
Improving prepare and build output

### PR DESCRIPTION
**Pull request by riknoll**
The purpose of this PR is to improve the quality and clarity of our logging (including verbose). Some things I tried to improve:

* Consistency of language (e.g. plugins are *added* to project and *installed* in platforms)
* Attribution of errors (e.g. announcing when hooks are being run and where they are coming from)
* Level of detail in vague logs and error messages
* Grammar, capitalization, punctuation, etc.

I will also be submitting PRs for cordova-android, cordova-ios, and cordova-windows to improve output on the platform side. I didn't touch any of the legacy platform code in this repo for that reason.

### CODE REVIEW

File: cordova-lib/src/hooks/HooksRunner.js [hunk 1]
```diff
@@ -141,6 +144,23 @@ function runScript(script, context) {
...
+        relativePath = path.join('plugins', script.plugin.id, script.path);
+    } else if (script.useModuleLoader) {
+        source = 'config.xml';
+        relativePath = path.normalize(script.path);
+    } else {
+        source = 'hooks directory';
+        relativePath = path.join('hooks', context.hook, script.path);
+    }
+
+    events.emit('verbose', 'Executing script found in ' + source + ' for hook "' + context.hook + '": ' + relativePath);
```
Comment (stevengill) [inline cordova-lib/src/hooks/HooksRunner.js]: Very handy!

**Comment by stevengill** [comment]
LGTM

**Comment by vladimir-kotikov** [comment]
LGTM :+1:

File: cordova-lib/src/cordova/plugin.js [hunk 2]
```diff
@@ -286,7 +286,7 @@ module.exports = function plugin(command, targets, opts) {
                                     var configXml = new ConfigParser(configPath);
                                     configXml.removePlugin(target);
                                     configXml.write();
-                                    events.emit('results', 'config.xml entry for ' +target+ ' is removed');
+                                    events.emit('results', 'config.xml entry for plugin ' + target + ' was removed');
```
Comment (jasongin) [inline cordova-lib/src/cordova/plugin.js]: What is the 'results' channel??

There is a similar log line for removing a platform from config.xml, only it comes before the change.

Comment (riknoll) [inline cordova-lib/src/cordova/plugin.js]: @jasongin updated! Switched it to use log

**Comment by jasongin** [comment]
:shipit:

**Comment by nikhilkh** [comment]
LGTM

**Comment by riknoll** [comment]
I'll merge whenever the Travis build finishes

**Comment by stevengill** [comment]
Travis taking foreverrrrr

**Comment by stevengill** [comment]
Passed! Lets merge this in @riknoll :)

**Comment by riknoll** [comment]
@stevengill  done!

**Pull request closed without merging**